spec/presenters/licence_finder_content_item_presenter_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe LicenceFinderContentItemPresenter do
+  let(:subject) { LicenceFinderContentItemPresenter.new }
```

Comment:
I'm on the fence about this, more recently I've tended to favour just referring to the object-under-test as 'subject' throughout. A peak at the filename gives you a pretty decent clue of what that might mean if you forget. But I don't feel particularly strongly either way.